packages/tooling/fast-tooling-react/src/form/controls/control.linked-data.tsx

```diff
@@ -240,8 +240,6 @@ class LinkedDataControl extends React.Component<
                 }
                 // Tab performs an autocompete if there is a single schema it can match to
             } else if (e.keyCode === keyCodeTab) {
```

Comment:
I wonder if this should conditionally `preventDefault()`. Not 100% sure but it seems that in cases when autocomplete is successful, moving focus may be un-intuitive.

Reply:
I agree, maybe you could simply move the `preventDefault()` into the `if` that checks for a matchedSchema below.

Reply:
Done and done, left a comment in the code. This now requires the user to hit enter to continue tab navigation though. I'm assuming that is the behavior we want in combination with tab completion functionality